hollow/src/main/java/com/netflix/hollow/api/producer/AbstractHollowMultiPublisher.java

```diff
@@ -0,0 +1,35 @@
+/*
+ *
+ *  Copyright 2018 Netflix, Inc.
+ *
+ *     Licensed under the Apache License, Version 2.0 (the "License");
+ *     you may not use this file except in compliance with the License.
+ *     You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *     Unless required by applicable law or agreed to in writing, software
+ *     distributed under the License is distributed on an "AS IS" BASIS,
+ *     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *     See the License for the specific language governing permissions and
+ *     limitations under the License.
+ *
+ */
+package com.netflix.hollow.api.producer;
```

Comment:
One more thing. For beta/experimental APIs I wanted to start appending `.experimental` to the package names. So:

```
package com.netflix.hollow.api.producer.experimental;
```

The benefits I see in moving to this naming convention:

* more clear that an API is in development
* when ready to make an API official we can include a set of breaking changes without actually breaking people's builds, by renaming the package and making any signature changes we want (in the case where we really like the class name and don't want to rename it)